client.go

```diff
@@ -234,6 +241,17 @@ func (c client) SendAppMetrics(m ContainerMetric) error {
 	return nil
 }
 
+func (c client) SendSpikeMetrics(m SpikeMetric) error {
```

Comment:
Wondering why yall added a different client method instead of just adding to the `SendAppMetrics` method? We already are sending a secondary envelope in that method that sends some CPU info

Reply:
The only reason we did that is the small refactoring in executor PR. There we separated emitting the current metric and the spike metric into two separate handlers and therefore we had to have two `SendXXXMetric` here. We could of course reconsider this implementation, maybe refactor executor handlers into builders or similar and then send all the data in a single shot